### ISSUE TITLE
presentation_definition may be provided in the RP's metadata

### ISSUE THREAD

**Issue by peppelinux** [openid/OpenID4VP#407]
I think that should enable the possibility to obtain the presentation_definition from the metadata instead that requiring it only in the authorization request

having the presentation_definition in the metadata allows:

1. the provisioning of metadata signed by a trusted third party, where the RP is allowed to use a specific presentation_definition
2. the possibility to apply dynamic metadata policy, according to the OpenID Federation policy language

Currently in the specs we have

> Presentation Definition JSON object MUST be sent using a presentation_definition parameter

and it is not explicit if the presentation_definition MUST be provided in the Authorization request or in the metadata

**Comment by danielfett** [comment]
I suspect that this can be closed since we don't have PE any longer...

**Comment by jogu** [comment]
I agree with Daniel - closing as no longer applicable now https://github.com/openid/OpenID4VP/pull/479 has been merged.
